modules/pubmaticBidAdapter.js

```diff
@@ -182,8 +184,11 @@ let biddersList = ['pubmatic'];
 const allBiddersList = ['all'];
 
 const vsgDomain = window.location.hostname;
-const getAndParseFromLocalStorage = key => JSON.parse(window.localStorage.getItem(key));
-let vsgObj = { [vsgDomain]: '' };
+let viewData;
+
+storage.getDataFromLocalStorage('viewability-data', val => {
+  viewData = JSON.parse(val);
```

Comment:
JSON.parse can throw an error when the given input is not a valid JSON(string), we should handle such scenario by setting a default value